packages/twenty-front/src/modules/object-record/object-filter-dropdown/utils/getOperandsForFilterType.ts

```diff
@@ -5,25 +5,40 @@ import { FilterType } from '../types/FilterType';
 export const getOperandsForFilterType = (
   filterType: FilterType | null | undefined,
 ): ViewFilterOperand[] => {
+  const emptyOperands = [
+    ViewFilterOperand.IsEmpty,
+    ViewFilterOperand.IsNotEmpty,
+  ];
+
+  const relationOperands = [ViewFilterOperand.Is, ViewFilterOperand.IsNot];
+
   switch (filterType) {
     case 'TEXT':
     case 'EMAIL':
     case 'FULL_NAME':
     case 'ADDRESS':
     case 'PHONE':
     case 'LINK':
-      return [ViewFilterOperand.Contains, ViewFilterOperand.DoesNotContain];
     case 'LINKS':
-      return [ViewFilterOperand.Contains, ViewFilterOperand.DoesNotContain];
+      return [
+        ViewFilterOperand.Contains,
+        ViewFilterOperand.DoesNotContain,
+        ...emptyOperands,
+      ];
     case 'CURRENCY':
```

Comment:
`CURRENCY` isEmpty filter returns an error 
![image](https://github.com/twentyhq/twenty/assets/29927851/8a791bb0-38e0-4cff-8dbd-d67adb8a6fcb)

Reply:
NUMBER / INTEGER isEmpty returns graphql error
![image](https://github.com/twentyhq/twenty/assets/29927851/b9d6c9e7-d0eb-4b3f-bec2-51b53737d162)

Reply:
RELATION isEmpty returns coffee break error
![image](https://github.com/twentyhq/twenty/assets/29927851/7a5d4d9b-260e-4656-a3c7-abcc71ea655e)

Reply:
@pacyL2K19

Reply:
Checking this asap @martmull 
Thanks for catching this